src/frontend/screens/MapScreen/MapLayers/RemoteDetectionAlertsLayer.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import MapboxGL from '@rnmapbox/maps';
+
+import {RemoteDetectionAlert} from '@comapeo/schema';
+import {FeatureCollection} from 'geojson';
+import {useRemoteDetectionAlerts} from '../../../hooks/server/remoteDetectionAlert';
+
+const LABEL_FILTER = [
+  'all',
+  [
+    'in',
+    '$type',
+    'Polygon',
+    'LineString',
+    'Point',
+    'MultiLineString',
+    'MultiPolygon',
+    'MultiPoint',
+  ],
+  ['has', 'alert_type'],
+  ['has', 'month_detec'],
+  ['has', 'year_detec'],
+];
+
+const POINT_FILTER = ['==', '$type', 'Point', 'MultiPoint'];
+
+const LINESTRING_FILTER = ['in', '$type', 'LineString', 'MultiLineString'];
+
+const POLYGON_STROKE_FILTER = ['in', '$type', 'Polygon', 'MultiPolygon'];
+
+const POLYGON_FILL_FILTER = ['in', '$type', 'Polygon', 'MultiPolygon'];
+
+export const RemoteDetectionAlertsMapLayer = () => {
+  const {data: alerts} = useRemoteDetectionAlerts();
+
+  if (!alerts) {
+    return null;
+  }
+
+  return (
+    <MapboxGL.ShapeSource
+      id="alerts-source"
+      shape={convertRemoteDetectionAlertsToFeatures(alerts)}>
+      {/* Symbol Layer for Labels */}
+      <MapboxGL.SymbolLayer
+        id="comapeo-alerts-label"
+        filter={LABEL_FILTER}
+        style={{
+          textField: [
+            'concat',
+            ['get', 'alert_type'],
```

Comment:
Needs to match key used in geojson properties

```suggestion
            ['get', 'alertType'],
```

Reply:
https://github.com/digidem/comapeo-mobile/pull/895/commits/313a10eafaa3eddf45bb9151022851dd480ef2f1